nnvm/tests/python/compiler/test_fold_axis.py

```diff
@@ -50,6 +50,52 @@ def check(shape, channels):
 
     check((2, 4, 10, 10), 2)
 
+def test_fold_axis_depthwise_conv():
+    def before(x, conv_weight, conv_bias, in_scale, out_scale, channels):
+        x = x * sym.expand_dims(in_scale, axis=1, num_newaxis=2)
+        y = sym.conv2d(x, conv_weight, conv_bias,
+                       channels=channels,
+                       kernel_size=(3, 3),
+                       padding=(1, 1),
+                       groups=54,
+                       name="depthiwise_conv")
+        y = sym.relu(y)
+        y = y * sym.expand_dims(out_scale, axis=1, num_newaxis=2)
+        return y
+
+    def expected(x, conv_weight, conv_bias, in_scale, out_scale, channels):
+        conv_weight = conv_weight * sym.expand_dims(out_scale, axis=1, num_newaxis=3)
+        conv_weight = conv_weight * sym.expand_dims(in_scale, axis=1, num_newaxis=3)
+        conv_bias = conv_bias * out_scale
+        y = sym.conv2d(x,
+                       conv_weight,
+                       conv_bias,
+                       channels=channels,
+                       kernel_size=(3, 3),
+                       padding=(1, 1),
+                       groups=54,
+                       name="depthiwise_conv")
+        y = sym.relu(y)
+        return y
+
+    # Before simplify
```

Comment:
This comment is in wrong place

Reply:
Thanks. I leverage the test_fold_axis_conv function, which comment at the top of `check` function. I think all of these comments this file should comment at the top of function `before`

Reply:
I find the `# Before simplify` comment at the top of `check` function is added by @tqchen firstly. Would you mind moving it at the top of `before` function or you have any thoughts?

Reply:
Yes, we should move

Reply:
Have moved the comments at the top of `before` function.